packages/api/src/routes/lightclient.ts

```diff
@@ -60,14 +62,16 @@ export type Api = {
 export const routesData: RoutesData<Api> = {
   getStateProof: {url: "/eth/v1/lightclient/proof/:stateId", method: "GET"},
   getCommitteeUpdates: {url: "/eth/v1/lightclient/committee_updates", method: "GET"},
-  getHeadUpdate: {url: "/eth/v1/lightclient/head_update/", method: "GET"},
+  getLatestHeadUpdate: {url: "/eth/v1/lightclient/latest_head_update/", method: "GET"},
+  getLatestFinalizedHeadUpdate: {url: "/eth/v1/lightclient/latest_finalized_head_update/", method: "GET"},
```

Comment:
```suggestion
  getLatestFinalizedHeadUpdate: {url: "/eth/v1/lightclient/latest_finalized_head_update", method: "GET"},
```

That last slash is a bit of a gotcha. Might be good to remove it from `latest_head_update/` at some point too. 😄